src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -34,7 +34,7 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS,
-                        PREFIX_REMARK, PREFIX_TAG);
+                         PREFIX_TAG, PREFIX_PROJECT);
```

Comment:
For standardization, I think `PREFIX_PROJECT` should come before `PREFIX_TAG` as follows:
`ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS,
                         PREFIX_PROJECT, PREFIX_TAG);`